nautilus_trader/adapters/interactive_brokers/client/client.py

```diff
@@ -390,6 +391,7 @@ async def _handle_disconnection(self) -> None:
             self._log.debug("`_is_ib_connected` unset by `_handle_disconnection`", LogColor.BLUE)
             self._is_ib_connected.clear()
 
+        self._last_disconnection_ns = self._clock.utc_now()
```

Comment:
This should resolve it
```suggestion
        self._last_disconnection_ns = self._clock.timestamp_ns()
```